frontend/lib/screens/lp_unauthorized_screen/lp_unauthorized_screen.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/material.dart';
+import 'package:workout_app/core/app_export.dart';
+
+class LpUnauthorizedScreen extends StatelessWidget {
```

Comment:
LoginPageUnathorizedScreen? thats the abbv for Lp?

Reply:
landing page – page that appears after splash